packages/xl-pdf-exporter/src/pdf/pdfExporter.test.tsx

```diff
@@ -160,8 +160,8 @@ describe("exporter", () => {
     );
 
     const transformed = await exporter.toReactPDFDocument(testDocument);
-    const view = render(transformed);
-    const str = prettyDOM(view.container, undefined, { highlight: false });
+    const str = reactElementToJSXString(transformed);
```

Comment:
what's the benefit of this library?

Reply:
Compared to `render` & `prettyDOM` it doesn't expect DOM/HTML-compatible JSX. Thus, it results in fewer warnings and fixes the `break` edge case I was facing (the `break` prop being removed from the snapshot due to incompatibility with DOM).